client/Components/Dog/Dog.jsx

```diff
@@ -9,7 +9,7 @@ import './dog.scss';
 const Dog = ({ dog }) => {
   return dog ? (
     <div id="dog-container">
-      <Link id="btn-back" className="btn-controls" to='/dogs'>Back to ALL DOGS</Link>
+      <Link id="btn-back" className="btn-controls" to='/dogs'><i className = 'left'></i>Back to ALL DOGS</Link>
```

Comment:
hi maria merged conflict. i fixed the floating button here. since it's pushing the Dog styles down.

Reply:
u need to accept incoming change for this one.